src/lib/style-properties.html

```diff
@@ -331,6 +334,9 @@
           // parsedSelector fallback for 'shady' css build
           selectorToMatch = rule.transformedSelector || rule.parsedSelector;
         }
+        if (isRoot && hostScope === 'html') {
```

Comment:
any reason not to always fallback from `transformedSelector` to `parsedSelector`